python/python/cwrap/prototype.py

```diff
@@ -148,7 +152,11 @@ def __call__(self, *args):
             self._resolved = True
 
         if self._func is None:
-            raise PrototypeError("Prototype has not been properly resolved!")
+            if self._allow_attribute_error:
+                raise NotImplementedError("Function:%s has not been properly resolved - missing library symbol?" % self.__name__)
+            else:
+                raise PrototypeError("Prototype not proper")
```

Comment:
I think it would be nicer to have the old message in the `PrototypeError` or something else that is a bit more informative.